pallets/staking/src/lib.rs

```diff
@@ -432,8 +442,6 @@ pub enum StakerStatus<AccountId> {
 /// A destination account for payment.
 #[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, RuntimeDebug)]
 pub enum RewardDestination<AccountId> {
-    /// Pay into the stash account, increasing the amount at stake accordingly.
-    Staked,
     /// Pay into the stash account, not increasing the amount at stake.
     Stash,
```

Comment:
Why `Staked` is removed in favor of `Stash`?

Reply:
Since the payout is in MNG, but the staking in liquidity token, we can't at this point support Staked.